r4/metadata/Ballerina.toml

```diff
@@ -4,17 +4,7 @@ name = "health.fhir.templates.r4.metadata"
 version = "1.0.0"
 authors = ["Ballerina"]
 keywords = ["Healthcare", "FHIR", "Metadata"]
-distribution = "2201.4.1"
+distribution = "2201.5.0"
 include = ["Cloud.toml", "Config.toml"]
-export = [
-    "health.fhir.templates.r4.metadata.config",
-    "health.fhir.templates.r4.metadata.constants",
-    "health.fhir.templates.r4.metadata.handlers",
-    "health.fhir.templates.r4.metadata.models",
-    "health.fhir.templates.r4.metadata.samples"
-]
-template = true
-repository="https://github.com/ballerina-platform/module-ballerinax-health.fhir.templates"
-
-[build-options]
-observabilityIncluded = true
+repository = "https://github.com/ballerina-platform/module-ballerinax-health.fhir.templates"
+template = true
```

Comment:
newline is missing. check other places as well

Reply:
Fixed in https://github.com/ballerina-platform/module-ballerinax-health.fhir.templates/commit/b794eeae55f66df0e5db97a64eb234d57775d32c